docs/app/Examples/elements/Button/Groups/ButtonGroupsExamples.js

```diff
@@ -0,0 +1,22 @@
+import React, {Component} from 'react';
+import ComponentExample from 'docs/app/Components/ComponentDoc/ComponentExample';
+import ExampleSection from 'docs/app/Components/ComponentDoc/ExampleSection';
+
+export default class ButtonStatesExamples extends Component {
```

Comment:
Classname here. It will be much easier to hear this from the linter, in the future..